src/song.ts

```diff
@@ -414,9 +412,19 @@ export class SongManager {
                 SongManager.cacheCleanTimeout = setTimeout(SongManager.clean, SongManager.cacheCleanTimeoutDuration);
             }
     }
+
+    static refreshMetadata() {
+        let date = Date.now();
+        SongManager.songs.forEach(async song => {
```

Comment:
Should you use a `for of` loop instead?

We had trouble with using `forOf` before with async